src/greenpak/i2c.py

```diff
@@ -185,3 +266,24 @@ def read(
                 )
                 print(str)
         return None
+
+    @override
+    def write_read(self, addr: int, wreq: bytearray, rbytes: int,
```

Comment:
I think you need to implement here, in the SMB implementation,  the three new methods of the interface, gp_read, gp_write, gp_scan, and delete the old ones read() and write()